app/views/collections/_show_document_list_menu.html.erb

```diff
@@ -1,7 +1,8 @@
 <div class="btn-group">
-  <button class="btn btn-default dropdown-toggle" data-toggle="dropdown">Select an action <span class="caret"></span>
+  <button class="btn btn-default dropdown-toggle" data-toggle="dropdown">Select an action<span class="caret"></span>
   </button>
   <ul class="dropdown-menu dropdown-menu-right">
+    <% if current_user && current_user.can?(:edit, document) %>
```

Comment:
Didn't think that was possible as test failed without current_user. Looking closer, I found the issue was in the test not the execution. Added `let(:ability) { instance_double("Ability") }` and `allow(controller).to receive(:current_ability).and_return(ability)` to fix.